### ISSUE TITLE
unable to register

### ISSUE THREAD

**Issue by imaginator** [project-isizwe/wifi-chat#27]
Looks like a new registration problem:

```
==> /var/log/wifi-chat-website.log <==
/opt/wifi-chat/src/database.js:74
  client.query(CREATE_ACCOUNT, record, callback)
         ^
TypeError: undefined is not a function
    at Object.addAccountRecord (/opt/wifi-chat/src/database.js:74:10)
    at /opt/wifi-chat/src/routes/account.js:65:12
    at null.callback (/opt/wifi-chat/src/database.js:44:5)
    at Query.handleReadyForQuery (/opt/wifi-chat/node_modules/pg/lib/query.js:80:10)
    at null.<anonymous> (/opt/wifi-chat/node_modules/pg/lib/client.js:158:19)
    at emit (events.js:129:20)
    at Socket.<anonymous> (/opt/wifi-chat/node_modules/pg/lib/connection.js:109:12)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
```

**Comment by imaginator** [comment]
![screenshot 2015-07-29 15 19 53](https://cloud.githubusercontent.com/assets/469572/8958479/53659698-3605-11e5-9a8c-1b93b522d3eb.png)

**Comment by lloydwatkin** [comment]
Fixed in 31c68b64d949f50a6ae33af257e60db5e7852904
